monai/data/dataset.py

```diff
@@ -354,6 +356,8 @@ def _cachecheck(self, item_transformed):
                         shutil.move(temp_hash_file, hashfile)
                     except FileExistsError:
                         pass
+        except PermissionError:  # project-monai/monai issue #3613
```

Comment:
Should we also mark the link of the original python issue here?

Thanks.

Reply:
ok, don't want to add too many comments in the code, let me put more details in ticket 3613 main texts...

Reply:
Sure, sounds good to me.
Thanks for the quick fix!